openai/openai.go

```diff
@@ -608,6 +645,17 @@ func (w *ChatWriter) writeResponse(data []byte) (int, error) {
 		}
 
 		if chatResponse.Done {
+			if w.streamUsage {
+				u := toUsage(chatResponse)
+				d, err := json.Marshal(ChatCompletionChunk{Choices: []ChunkChoice{}, Usage: &u})
```

Comment:
Pushed an additional commit setting choices to empty slices instead of nil, the former serializes as `[]`, the latter `null`. I was testing with python openai library without issue but found this wasn't working with JS openai library.